core/src/main/java/org/jdbi/v3/core/mapper/reflect/ConstructorMapper.java

```diff
@@ -44,10 +47,7 @@
 import static java.lang.String.format;
 
 import static org.jdbi.v3.core.mapper.reflect.JdbiConstructors.findFactoryFor;
-import static org.jdbi.v3.core.mapper.reflect.ReflectionMapperUtil.addPropertyNamePrefix;
-import static org.jdbi.v3.core.mapper.reflect.ReflectionMapperUtil.anyColumnsStartWithPrefix;
-import static org.jdbi.v3.core.mapper.reflect.ReflectionMapperUtil.findColumnIndex;
-import static org.jdbi.v3.core.mapper.reflect.ReflectionMapperUtil.getColumnNames;
+import static org.jdbi.v3.core.mapper.reflect.ReflectionMapperUtil.*;
```

Comment:
please fix that. :-) We don't do wildcard imports. There are code style file for the most common IDEs in the ide-support folder (or at https://github.com/jdbi/jdbi/tree/master/ide-support)

Reply:
Yes I guess my IDE did this. I will have a look at this later

Reply:
Fixed